packages/ui-extensions/src/surfaces/point-of-sale/api/navigation-api/navigation-api.ts

```diff
@@ -1,12 +1,14 @@
 export interface NavigationApiContent {
-  /** Navigate to a route in current navigation tree.
+  /**
+   * Using `(screenName: string, params?: any) => void` navigates to a route in current navigation tree.
    * Pushes the specified screen if it isn't present in the navigation tree, goes back to a created screen otherwise.
    * @param screenName the name of the screen you want to navigate to.
    * @param params the parameters you want to pass to that screen.
    */
   navigate(screenName: string, params?: any): void;
```

Comment:
This is forward looking as we could throw errors in the future. Also if you print navigate func, it is always promise cause of the rpc layer. Should be safe cause void -> Promise< void > won't affect extension code since they aren't really doing anything with a result.

```suggestion
  navigate(screenName: string, params?: any): Promise<void>;
```